agasc/scripts/update_mag_supplement.py

```diff
@@ -163,14 +163,17 @@ def main():
 
     star_obs_catalogs.load(args.stop)
 
-    # set the list of AGASC IDs from file if specified. If not, it will include all.
-    agasc_ids = []
-    if args.agasc_id_file:
-        with open(args.agasc_id_file, "r") as f:
-            agasc_ids = [int(line.strip()) for line in f.readlines()]
+    if "agasc_ids" in file_args:
+        agasc_ids = file_args["agasc_ids"]
+    else:
+        # set the list of AGASC IDs from file if specified. If not, it will include all.
+        agasc_ids = []
+        if args.agasc_id_file:
+            with open(args.agasc_id_file, "r") as f:
+                agasc_ids = [int(line.strip()) for line in f.readlines()]
 
-    # update 'bad' and 'obs' tables in supplement
-    agasc_ids += update_supplement.update(args)
+        # update 'bad' and 'obs' tables in supplement
+        agasc_ids += update_supplement.update(args)
```

Comment:
Is this new indent on purpose?  I would think you'd still want to do the update_supplement.update(args) even if you've been given list of agasc_ids in file_args, but maybe I'm missing a logical change here.

Reply:
Aha! good eye. Yes, I changed this on purpose, but I think you have a point.

Note that there is another change. This change in indentation goes with the `if "agasc_ids" in file_args:` statement, which was not there to begin with, so everything after `agasc_ids = []` is getting extra indentation.

This adds a feature that I did not document in the PR description. You can make a `call_args.yml` file by hand which includes the `agasc_ids` key, with a list of AGASC IDs. Note that there is no `--agasc-ids` command line argument, this is a feature of the `call_args.yml` file.

The truth is that we have never used the update_supplement part together with the update magnitudes part.

Reply:
I can change the indentation back. I don't think I have strong opinion on that, and maybe it is best to leave it as it  was to make sure the same process is run.